crates/oci-distribution/src/client.rs

```diff
@@ -397,7 +429,45 @@ mod test {
     use super::*;
     use std::convert::TryFrom;
 
-    const HELLO_IMAGE: &str = "webassembly.azurecr.io/hello-wasm:v1";
+    const HELLO_IMAGE_NO_TAG: &str = "webassembly.azurecr.io/hello-wasm";
```

Comment:
This currently breaks e2e tests because `webassembly.azurecr.io/hello-wasm:latest` is not a valid reference. This can be removed, but we'll have better coverage if we add a `latest` tag pointing to the same digest as `v1`.

Sidebar: implementing a mock registry endpoint to avoid going over the network during test execution would probably be a good idea.

Reply:
done